tests/test_json.ml

```diff
@@ -0,0 +1,60 @@
+let read_file fname =
+  (* the tests are run from the _build/default/tests directory *)
+  Fpath.(v ".." / ".." / ".." / "tests" / fname)
```

Comment:
There's a better way to deal with this in dune directly. The json files could be declared as a dependency of the test alias if we want to stick to this. That way they'll be copied to the `_build` dir as well and if they are ever modified or moved, dune will behave properly. The `../` thing might not survive even a slight change in the project structure, it's a bit fragile and not always very easy to figure out when it breaks.

I'm wondering if it wouldn't be better to write them as raw strings in a separate module though. That would spare us the need to override the default dune actions.

What do you think?

Reply:
I agree, I added the `*_response.ml` files to the test directory.